python/ray/_private/function_manager.py

```diff
@@ -124,15 +118,37 @@ def compute_collision_identifier(self, function_or_class):
         # Return a hash of the identifier in case it is too large.
         return hashlib.sha1(collision_identifier.encode("utf-8")).digest()
 
+    def check_local(self,module_name,function_class_name):
```

Comment:
better name this `load_function_or_class_from_local`, as it returns the loaded object. 
`check_sth` usually returns a boolean
```
    def load_function_or_class_from_local(self, module_name, function_or_class_name):
```